guides/common/modules/proc_configuring-provisioning-callback-for-a-host.adoc

```diff
@@ -47,8 +47,9 @@ For more information about job templates, see http://docs.ansible.com/automation
 |====
 |Name |Value |Description
 |`ansible_tower_provisioning` |true |Enables Provisioning Callback.
-|`ansible_tower_fqdn` |_controller.example.com_ |The fully qualified domain name (FQDN) of your {awx}.
-Do not add `https` because this is appended by {Project}.
+|`ansible_tower_api_url` |_https://controller.example.com/api/controller/v2_ | Defines the URL of your {awx}, including the required API path.
+For older versions of {awx}, use `/api/v2` instead of `/api/controller/v2`.
```

Comment:
Older than which version?

Reply:
AAP 2.5 - See the discussion here: https://github.com/theforeman/foreman-documentation/pull/3720#discussion_r1987691773.  

**Summary:** These changes are required for AAP 2.5+. If you're using an older AAP version or AWX (instead of AAP), these changes don't apply. In that case, you'll need to manually set the `ansible_tower_api_url` endpoint to `/api/v2` instead of `/api/controller/v2`.